### ISSUE TITLE
I can’t find the FaceID setting anywhere in my settings tab but when I gesture up on the lock scree it say “face ID required”  I’m on and iPhone SE 2020 with iOS 13+ please help.

### ISSUE THREAD

**Issue by tjay2three** [ryannair05/Little11#124]
I can’t find the FaceID setting anywhere in my settings tab but when I gesture up on the lock scree it say “face ID required”  I’m on and iPhone SE 2020 with iOS 13+ please help.

__Originally posted by @sirsuperwimp in https://github.com/ryannair05/Little11/issues/103__

**Comment by akaROOSTA** [comment]
you can use the face ID function on iphone SE 2020 ????

i'm on ios 13.5 iPhone SE 2020 also.
teach me the ways dude i wanna use face unlock too!!!!

**Comment by akaROOSTA** [comment]
Yeah I don't this our iPhone generation doesn't come woth that type of hardware man. I've sought out multiple tweaks that have this feature and none of them end up working out
